core/computed/metrics/interactive.js

```diff
@@ -86,11 +86,13 @@ class Interactive extends NavigationMetric {
    * @return {{cpuQuietPeriod: TimePeriod, networkQuietPeriod: TimePeriod, cpuQuietPeriods: Array<TimePeriod>, networkQuietPeriods: Array<TimePeriod>}}
    */
   static findOverlappingQuietPeriods(longTasks, networkRecords, processedNavigation) {
-    const FcpTsInMs = processedNavigation.timestamps.firstContentfulPaint / 1000;
+    const minTime = processedNavigation.timestamps.largestContentfulPaint !== undefined ?
+      processedNavigation.timestamps.largestContentfulPaint / 1000 :
+      processedNavigation.timestamps.firstContentfulPaint / 1000;
```

Comment:
Not sure if it matters since this is TTI we're talking about (but that applies to making any changes at all, so... :), but seems bad to have this fallback? Makes the metric harder to reason about, discontinuous for an otherwise identical trace depending on if LCP was captured or not, and diverges from the lantern version, which throws if there was no LCP.

Reply:
I agree that the fallback in TTI can introduce inconsistencies. While it might seem like a minor detail, the discontinuity can complicate analysis and make it difficult to accurately compare performance across different scenarios. Perhaps we could explore alternative approaches, such as using a default value: If LCP is not captured, we could set TTI to a default value (e.g., the maximum possible value) to indicate that the metric is incomplete or even introducing a new metric: We might consider creating a separate metric that specifically measures time to first interaction without relying on LCP.

Reply:
So alternatively, we can throw if there's no LCP

and ignore this FCP fallback